docs/application/web/api/4.0/device_api/mobile/tizen/calendar.html

```diff
@@ -2077,7 +2077,8 @@ <h4>Methods</h4>
             </p>
 <div class="description">
             <p>
-If the <em>watchId </em>argument is valid and corresponds to a subscription already in place, the watch process must immediately stop and no further callbacks must be invoked. If the <em>watchId </em>argument is not valid or does not correspond to a valid subscription, the method should return without any further action.
+If the <em>watchId </em>argument is valid and corresponds to a subscription already in place, the watch process must immediately stop and no further callbacks must be invoked.
+If the <em>watchId </em>argument is not valid or does not correspond to a valid subscription, the method should return without any further action.
```

Comment:
Just for check : This line separating will not work in the html file.
To separate the lines, \<br\> or \<p\> should be used.

Reply:
It was added just avoiding very long line in source file. Separating lines was not our intention here. "This line separating will not work in the html file." - it is exactly what we expected.